src/data/addAppVersionMedia.js

```diff
@@ -50,7 +50,7 @@ const paramSchema = joi
 const addAppVersionMedia = async (params, knex, transaction) => {
     const validation = paramSchema.validate(params)
 
-    if (validation.error !== null) {
+    if (validation.error !== undefined) {
```

Comment:
I assume there's a reason we simply do not `if (validation.error)` ?

Reply:
I just changed it to this due to the ease of find and replacing it. I'm sure we could do that as well.